workflow/setup_expt.py

```diff
@@ -61,7 +61,7 @@ def fill_COMROT_cycled(host, inputs):
 
     do_ocean = do_ice = do_med = False
 
-    if inputs.app in ['S2S', 'S2SW']:
+    if 'S2S' in inputs.app:
```

Comment:
Not sure why one would rely on an implicit wildcard when the explicit list is clearer.
But, `in` allows it, so I can't argue.

Reply:
This (mostly) mirrors how `config.base` does it (config.base technically only matches `S2S` at the beginning of the string, but that is the only place we put it).

Reply:
Works for me.